games/datingsim/resources/initial_prompts/initialprompt_playerA.template

```diff
@@ -1,23 +1,26 @@
-On each further turn you can react to what the other player tells you. For this you can either continue the conversation if you have not met an agreement yet or are not happy with the proposal, or end the game by agreeing to the proposal.
+On each further turn you can react to what the other player tells you. For this you can either continue the conversation if you have not met an agreement yet or are not happy with the proposal, or end the game by agreeing to the proposal. 
+
+There are four types of agreement: "Agreement on Time", "Agreement on Location", "Agreement on Action", "Overall Agreement". You and the other player needs to find an agreement on each of these types.
+You can only choose "Overall Agreement" after you and your date manage to settle an agreement for the other three types (Time, Location, Action), otherwise the game is evaluated as lost
 
 Your answer has to consist ONLY of the following information and follow the given template:
 
 [reason] Your reason for your decision. [end]
 
-[sentiment] Choose one of the following: "Found Agreement", "Continue Conversation" [end]
+[sentiment] Choose one of the following: "Continue Conversation", "Agreement on Time", "Agreement on Location", "Agreement on Action", "Overall Agreement" [end]
 
 [response] Your response to the other player. This can include your sentiment, opinion, alternative. [end]
 
 Your answer for [response] must NOT exceed 100 tokens.
 
-The game ends when both you and the other player reply with the sentiment "Found agreement", when you run out of turns to communicate or you do not follow the template.
-Be careful with your sentiment "Found agreement". Unless you and the other player say "Found agreement" at the same in [sentiment], the game is evaluated as lost
+The game ends when both you and the other player reply with the sentiment "Overall Agreement", when you run out of turns to communicate or you do not follow the template.
+Be careful with your sentiment "Overall Agreement". Unless you and the other player say "Found agreement" at the same in [sentiment], the game is evaluated as lost
 
 This is your character: $charsheet_a
 
 This is the character of the other player: $charsheet_b
 
-Remember, your goal is to find an agreement for your first date while staying in character.
+Remember, your goal is to find the overall agreement for your first date while staying in character.
 
 Start the game with the reason, your sentiment and the opening phrase.
```

Comment:
Just a tiny add of endline here (add an enter to the end of the file coz otherwise github cries)